### ISSUE TITLE
Dynamically fetch import paths

### ISSUE THREAD

**Issue by reuvenharrison** [ribice/glice#10]
When the dependency isn't on a well know repo (like github) there is an option to dynamically fetch the import path over https/http and discover where the code resides from a <meta> tag in the HTML

This is described here:
https://golang.org/cmd/go/#hdr-Remote_import_paths

For example, if you have a dependency on go.uber.org/zap, you can run:
```curl go.uber.org/zap```
And see that the repo is actually on github:
```
<!DOCTYPE html>
<html>
    <head>
        <meta name="go-import" content="go.uber.org/zap git https://github.com/uber-go/zap">
        <meta name="go-source" content="go.uber.org/zap https://github.com/uber-go/zap https://github.com/uber-go/zap/tree/master{/dir} https://github.com/uber-go/zap/tree/master{/dir}/{file}#L{line}">
        <meta http-equiv="refresh" content="0; url=https://godoc.org/go.uber.org/zap">
    </head>
    <body>
        Nothing to see here. Please <a href="https://godoc.org/go.uber.org/zap">move along</a>.
    </body>
</html>
```

**Comment by reuvenharrison** [comment]
I wrote code to do this - need access your repo to create a branch and a PR

**Comment by ribice** [comment]
Excellent. You can fork the repo, make the changes there and then create a PR. That's how it's usually done, no need for repo access.

**Comment by reuvenharrison** [comment]
https://github.com/Tufin/glice/pull/1

**Comment by ribice** [comment]
You need to create PR on this repo, not your own. Click on this repo, 'Pull Request' and then select branch from your fork.

**Comment by ribice** [comment]
Resolved with #11
